source/renderer/app/features/discreet-mode/replacers/discreetWalletTokenAmount.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import BigNumber from 'bignumber.js';
+import type { AssetMetadata } from '../../../api/assets/types';
+import { formattedTokenWalletAmount } from '../../../utils/formatters';
+import type { DiscreetValueReplacer } from '../types';
+
+export type DiscreetWalletTokenAmountProps = {
+  amount: BigNumber,
+  metadata?: ?AssetMetadata,
+  decimals: ?number,
+  isShort?: boolean,
+};
+
+export const discreetWalletTokenAmount: DiscreetValueReplacer = ({
```

Comment:
We are missing unit tests for this replacer

Reply:
It's covered by `<DiscreetTokenWalletAmount/>` test cases:

https://github.com/input-output-hk/daedalus/pull/2756/files#diff-70dd55c2f2cc99ed589490581ed9ea3fbf2613561a96012194d16a5c00594579R85

Reply:
Yeah probably good enough although it's an integration test 😉 but in this case, i guess it makes sense because it's really only used as a component prop 👍